lib/atscppapi/examples/helloworld/HelloWorldPlugin.cc

```diff
@@ -20,16 +20,23 @@
 #include <iostream>
 #include <atscppapi/GlobalPlugin.h>
 #include <atscppapi/PluginInit.h>
-
+#include <../../../../proxy/api/ts/ts.h>
```

Comment:
Yes, but I think that's a reasonable long term goal. I think it's better than making CPP API plugins have to deal with `ts.h` - avoiding that seems a major goal of this library.